enterprise_access/apps/subsidy_request/models.py

```diff
@@ -53,6 +56,16 @@ class SubsidyRequest(TimeStampedModel, SoftDeletableModel):
         max_length=255
     )
 
+    course_partners = JSONField(
+        blank=True,
+        null=True,
+        load_kwargs={'object_pairs_hook': collections.OrderedDict},
+        dump_kwargs={'indent': 4, 'cls': JSONEncoder, 'separators': (',', ':')},
+        help_text=_(
+            "List of course partner dictionaries."
+        )
+    )
+
```

Comment:
The fact that we now have 1 field identifying a course and 2 fields holding metadata about a course, all on a model called `SubsidyRequest`, smells fishy.  It's outside the scope of this PR, but we should consider replicating the content metadata we need into a new model.

Reply:
I agree, those 2 fields don't really belong in this model :/. It does feel like a decent size scope increase though, we'd probably want to update the serializer to reflect the nested structure, update consumers, etc. Should this be a quick tech debt follow-up?.

Reply:
@iloveagent57 @long74100 I agree it feels out of scope for this particular PR at this point. Let's tackle it as a follow-up? Also, curious - do we have a similar situation like this in `CustomerAgreement` in license-manager since it's storing 1 field to identify a enterprise UUID and 2 fields holding metadata about the customer?

Is the difference that there's a 1:1 mapping between CustomerAgreement <-> EnterpriseCustomer vs. many SubsidyRequest <-> 1 course?

Reply:
Filed ticket to capture these thoughts: https://openedx.atlassian.net/browse/ENT-5776

Reply:
I think we do have a similar situation in CustomerAgreement. The 1:1, 1:n mapping difference doesn't matter too much imo.